src/presentation/react/core/components/sync-wizard/common/SummaryStep.tsx

```diff
@@ -216,32 +228,49 @@ export const SummaryStepContent = (props: SummaryStepContentProps) => {
                 </ul>
             </LiEntry>
 
-            {_.keys(metadata).map(metadataType => {
-                //@ts-ignore
-                const modelByMetadataType = api.models[metadataType];
-                if (!modelByMetadataType) {
-                    console.warn(`Metadata type "${metadataType}" not supported in d2-api`);
-                    return null;
-                }
-                const itemsByType = metadata[metadataType as keyof MetadataEntities] || [];
+            {_(metadata)
+                .keys()
+                .concat(syncRule.metadataSyncAll)
+                .uniq()
+                .sort()
+                .value()
+                .map(metadataType => {
+                    //@ts-ignore
+                    const modelByMetadataType = api.models[metadataType];
+                    if (!modelByMetadataType) {
+                        console.warn(`Metadata type "${metadataType}" not supported in d2-api`);
```

Comment:
Not important here, it's just a message, but we won't like to find references to d2-api (data layer) in the presentation layer.